examples/events/rpiTouchExample/src/ofApp.cpp

```diff
@@ -0,0 +1,123 @@
+#include "ofApp.h"
+
+//--------------------------------------------------------------
+void ofApp::setup(){	 
+
+	vagRounded.load("vag.ttf", 32);
+	ofBackground(50,50,50);	
+}
+
+//--------------------------------------------------------------
+void ofApp::update(){
+
+}
+
+//--------------------------------------------------------------
+void ofApp::draw(){
+	
+	timeString = "time: " + ofGetTimestampString("%H:%M:%S") + "\nelapsed time: " + ofToString(ofGetElapsedTimeMillis());
+		
+	ofSetHexColor(0xffffff);
+	vagRounded.drawString(timeString, 98,98);
+	
+	ofSetColor(255,122,220);
+	vagRounded.drawString(timeString, 100,100);
+
+
+    for(auto finger: touches){
+        if(finger.second != nullptr){
+            ofDrawCircle(*finger.second, 50);
+        }
+    }
+}
+
+//--------------------------------------------------------------
+void ofApp::keyPressed  (int key){
+
+	
+}
+
+//--------------------------------------------------------------
+void ofApp::keyReleased(int key){
+    
+
+}
+
+//--------------------------------------------------------------
+void ofApp::mouseMoved(int x, int y ){
+
+}
+
+//--------------------------------------------------------------
+void ofApp::mouseDragged(int x, int y, int button){
+
+}
+
+//--------------------------------------------------------------
+void ofApp::mousePressed(int x, int y, int button){
+
+}
+
+//--------------------------------------------------------------
+void ofApp::mouseReleased(int x, int y, int button){
+
+}
+
+//--------------------------------------------------------------
+void ofApp::mouseEntered(int x, int y){
+
+}
+
+//--------------------------------------------------------------
+void ofApp::mouseExited(int x, int y){
+
+}
+
+//--------------------------------------------------------------
+void ofApp::touchDown(int x, int y, int id){
+    if(touches[id] == nullptr){
+        touches[id] = new ofVec2f(x, y);
+    }else{
+        touches[id]->set(x, y);
+    }
+    
+}
+
+//--------------------------------------------------------------
+void ofApp::touchMoved(int x, int y, int id){
+    if(touches[id] == nullptr){
+        touches[id] = new ofVec2f(x, y);
+    }else{
+        touches[id]->set(x, y);
+    }
```

Comment:
I think a better approach here would be to use a std::map<int, ofVec2f> and then just have in the touchDown / touchMoved: 

` touches[id] = ofVec2f(x, y);`

then in touch up:

`touches.erase(id);` 

and draw would just be:

```
     for(auto finger: touches){
            ofDrawCircle(finger.second, 50);
     }
```